mods/creative/inventory.lua

```diff
@@ -1,5 +1,7 @@
 creative = {}
-local player_inventory = {}
+local creative_mode = minetest.setting_getbool("creative_mode")
+local player_inventory, original_list = {}, {}
+local max, floor, ceil, sort = math.max, math.floor, math.ceil, table.sort
```

Comment:
this code ain't performance critical thus this replacement is not required.
If it's more readable is surely a matter of taste but I'd prefere math.max to the simplified variant.